easybuild/easyblocks/c/cuda.py

```diff
@@ -78,6 +91,35 @@ def install_step(self):
         #overriding maxhits default value to 300 (300s wait for nothing to change in the output without seeing a known question)
         run_cmd_qa(cmd, qanda, std_qa=stdqa, no_qa=noqanda, log_all=True, simple=True, maxhits=300)
 
+
+    def post_install_step(self):
+        # Wrapper script definition
+        wrapper = """#!/bin/sh
```

Comment:
defining the wrapper script as a constant outside of the `EB_CUDA` class would be nicer